app/app.rb

```diff
@@ -0,0 +1,43 @@
+require 'sinatra/base'
+require_relative './data_mapper_setup'
+
+class Chitter < Sinatra::Base
+  set :views, proc {File.join(root,'..','/app/views')}
+  enable :sessions
+  set :session_secret, 'super secret'
+
+  helpers do
+    def current_user
+      User.get(session[:user_id])
+    end
+  end
+
+
+  get '/peeps' do
+    @peeps = Peep.all
+    erb :'peeps/index'
+  end
+
+  get '/peeps/new' do
+    erb :'peeps/new_peep'
+  end
+
+  post '/peeps' do
+  Peep.create(message: params[:message])
+  redirect '/peeps'
+end
```

Comment:
Lint/BlockAlignment: `end` at 28, 0 is not aligned with `post '/peeps' do` at 25, 2